openreview/conference/builder.py

```diff
@@ -1169,13 +1179,20 @@ def is_release_authors(is_note_accepted):
 
 class SubmissionStage(object):
 
+    class Readers(Enum):
+        EVERYONE = 0
+        AREA_CHAIRS = 1
+        AREA_CHAIRS_ASSIGNED = 2
+        REVIEWERS = 3
+        REVIEWERS_ASSIGNED = 4
+
     def __init__(
             self,
             name='Submission',
             start_date=None,
             due_date=None,
             second_due_date=None,
-            public=False,
```

Comment:
Shouldn't we still keep this public here? We keep using self.public in some places

Reply:
I'm defining the property below, I would like to remove it so we start using `readers`, I'm keeping the `public` property in the builder anyway.